<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreasureWidget</class>
 <widget class="QWidget" name="TreasureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreasureWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="collectableLayout" stretch="1,1,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="itemsBox">
       <property name="title">
        <string>Items</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="itemsRandom">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="itemsRandomSkeletonKeys">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>+Random Skeleton Keys</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="itemsRandomByCategory">
          <property name="text">
           <string>Random By Category</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="valuablesBox">
       <property name="title">
        <string>Valuables</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="valuablesRandom">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="valuablesRandomByCategory">
          <property name="text">
           <string>Random By Category</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="valuablesRandomUsableValuables">
          <property name="text">
           <string>+Random Usable Valuables</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="equipmentsBox">
       <property name="title">
        <string>Equipments</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
        <item>
         <widget class="QCheckBox" name="equipmentsRandom">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="equipmentsRandomByCategory">
          <property name="text">
           <string>Random By Category</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="battleBox">
     <property name="title">
      <string>Battle</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="battleShuffle">
        <property name="text">
         <string>Shuffle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="battleShuffleSkeletonKeys">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>+Shuffle Skeleton Keys</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
